setup.cfg

```diff
@@ -0,0 +1,65 @@
+[metadata]
+name = plotman
+version = file: VERSION
+author = Eric Altendorf
+author-email = PLACEHOLDER
+home-page = https://github.com/ericaltendorf/plotman
+description = Chia plotting manager
+long-description = file: README.md
+long_description_content_type = text/markdown
+license = Apache 2.0
+license-file = LICENSE
+platform = any
+keywords = chia, blockchain, automation, process management
+classifiers =
+    Development Status :: 3 - Alpha
+    Environment :: Console :: Curses
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Information Technology
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Topic :: Office/Business :: Scheduling
```

Comment:
I think this is maybe for other forms of scheduling.  `:]`

Reply:
I definitely meant it in this sense:
![image](https://user-images.githubusercontent.com/31668498/113236258-88900900-9272-11eb-8f89-488b1a44a619.png)